### ISSUE TITLE
Reorder documentation

### ISSUE THREAD

**Issue by stereoit** [szw/vim-ctrlspace#158]
Hello, this plugin is really cool, but the learning curve is quite high. Would it be possible to perhaps reorder the inline documentation (I just recently noticed ? works different in every section)?

For example have all "Move" actions together, or group item related things together (buffer operations, tab operations).

**Comment by szw** [comment]
Yeah, the documentation items could have a priority or/and group attributes. We would use them later while sorting. The order has to be maintain programmatically (not hardcoded) since it's possible to provide custom docs for custom user functions.
